src/styles/reset.css

```diff
@@ -0,0 +1,8 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  display: flex;
```

Comment:
Why do you need it? Making your block dependent on outer styles isn't a good idea. Look, if I turn it off, something breaks
![image](https://user-images.githubusercontent.com/98764641/197177290-a395c7b4-0812-4e72-8352-b2c7cee7435a.png)